roles/kubernetes/master/templates/secrets_encryption.yaml.j2

```diff
@@ -0,0 +1,11 @@
+kind: EncryptionConfig
+apiVersion: v1
+resources:
+  - resources:
+    - secrets
+    providers:
+    - {{ encryption_algorithm }}:
+        keys:
+        - name: key
+          secret: {{ encrypt_token }}
```

Comment:
it should be `base64` encode from encryption token, not a plain token

Reply:
You are right! Fixed